compliance_wi_compliance.md

```diff
@@ -1,10 +1,8 @@
 Wisconsin Compliance
 ====================
+M-RETS is the designated tracking system for the state of Wisconsins RPS. In order for certificates to be used for WI Compliance the generator associated with the the certitifcates must have the necessary Wisconsin eligibilities applied to it. Generators seeking "Wisconsin: Bankable: eligiblity must submit the necessary approval documents from the Wisconisn Public Service Commission.
 
-Wisconsin Compliance Assistance - Link to PSC Website
+Wisconsin Three Year Hydro Average Information
+https://www.mrets.org/wp-content/uploads/2023/06/M-RETS-Overview-of-WI-Three-Year-Average-6-5-23-1.pdf
 
-Below is the link you can use to access the Wisconsin specific training where we reviewed RRC retirements and how they should occur for this specific 2017 Wisconsin RPS Compliance Retirements.
-
-<https://zoom.us/recording/play/HcnGHua4A9V94xOZgI2zhxW_60TLmhJKMOyRZvyMrvBRx8GVOO3lfi1xparFhbfF>
-
-If you would like further information on Wisconsin Compliance statutes and regulations, please see the Wisconsin PSC website at <https://psc.wi.gov/Pages/Programs/RpsCompliance.aspx>
+If you would like further information on Wisconsin Compliance statutes and regulations, please see the Wisconsin PSC website at <https://psc.wi.gov/Pages/Programs/RpsCompliance.aspx> and https://docs.legis.wisconsin.gov/statutes/statutes/196/378
```

Comment:
For more information on Wisconsin..., see...

[throughout text, you can kill instances of "please"]